ledger/hash_stores/file_hash_store.py

```diff
@@ -95,15 +96,19 @@ def readNodes(self, startpos, endpos):
 
     @property
     def leafCount(self) -> int:
-        return self.leavesFile.dbFile.seek(0, 2) // self.leafSize
+        return self.leavesFile.db_file.seek(0, 2) // self.leafSize
```

Comment:
db_file is a public property as of now for the file storage implementation and used in many places. Let's change it later.